apps/ledger-live-mobile/src/components/Nft/NftList.tsx

```diff
@@ -51,6 +52,30 @@ export function NftList({ data }: Props) {
     multiSelectModeEnabled,
   } = useNftList({ nftList: data });
 
+  const onPressMultiselect = useCallback(() => {
+    track("button_clicked", {
+      button: "Hide NFTs",
+      screen: ScreenName.WalletNftGallery,
+    });
+    triggerMultiSelectMode();
+  }, [triggerMultiSelectMode]);
+
+  const onPressHide = useCallback(() => {
+    track("button_clicked", {
+      button: "Multi Hide NFTs",
+      screen: ScreenName.WalletNftGallery,
+    });
+    onClickHide();
+  }, [onClickHide]);
+
+  const onCancelHide = useCallback(() => {
+    track("button_clicked", {
+      button: "Cancel  Hide NFTs",
+      screen: ScreenName.WalletNftGallery,
+    });
+    exitMultiSelectMode();
+  }, [exitMultiSelectMode]);
+
```

Comment:
I would have put this in the hook to keep consistency on the separation of the "logic" and the UI.

Reply:
But LGTM